Cannot find module node_modules/lodash-es/get

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/node_modules/lodash-es/get' imported from /node_modules/bknd/dist/ui/chunks/chunk-ULQ54IAG.js
Did you mean to import "lodash-es/get.js"?
    at finalizeResolution (node:internal/modules/esm/resolve:265:11)
    at moduleResolve (node:internal/modules/esm/resolve:933:10)
    at defaultResolve (node:internal/modules/esm/resolve:1157:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:227:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:87:39)
    at link (node:internal/modules/esm/module_job:86:36)


I get this error locally when I try hitting the /admin route in the browser.

--- a/app/src/ui/components/form/json-schema/JsonSchemaValidator.ts
+++ b/app/src/ui/components/form/json-schema/JsonSchemaValidator.ts
@@ -12,7 +12,7 @@ import type {
    ValidatorType
 } from "@rjsf/utils";
 import { toErrorSchema } from "@rjsf/utils";
-import get from "lodash-es/get";
+import { get } from "lodash-es";
 
 function removeUndefinedKeys(obj: any): any {
    if (!obj) return obj;

--- a/app/src/ui/components/form/json-schema/fields/MultiSchemaField.tsx
+++ b/app/src/ui/components/form/json-schema/fields/MultiSchemaField.tsx
@@ -14,9 +14,7 @@ import {
    getWidget,
    mergeSchemas
 } from "@rjsf/utils";
-import get from "lodash-es/get";
-import isEmpty from "lodash-es/isEmpty";
-import omit from "lodash-es/omit";
+import { get, isEmpty, omit } from "lodash-es";
 import { Component } from "react";
 import { twMerge } from "tailwind-merge";
 import { Label } from "../templates/FieldTemplate";

